partiql-lang/src/test/kotlin/org/partiql/lang/syntax/PartiQLParserTest.kt

```diff
@@ -920,6 +920,36 @@ class PartiQLParserTest : PartiQLParserTestBase() {
         "(parameter 1)"
     )
 
+    @Test
+    fun currentUserUpperCase() = assertExpression(
+        "CURRENT_USER",
+        "(session_attribute CURRENT_USER)"
+    )
+
+    @Test
+    fun currentUserMixedCase() = assertExpression(
+        "CURRENT_user",
+        "(session_attribute CURRENT_user)"
+    )
```

Comment:
Not sure if I want to keep this can of worms open -- so only if you'd like...

Is there a good reason to preserve the case of the keyword as it was entered?  
This does not happen for other keywords (SELECT, WHERE).
I understand `session_attribute` as a generic implementation to use instead of several more concrete AST nodes for CURRENT_USER and friends. With the latter, case would not be preserved either.